src/tools/cephfs/cephfs-shell

```diff
@@ -157,8 +157,13 @@ def get_all_possible_paths(pattern):
     for pattern in patterns:
         for path in paths:
             paths.extend(glob(path, pattern))
-    return [path for path in paths if fnmatch.fnmatch(path,
-            os.path.join(cephfs.getcwd(), complete_pattern))]
+    pat = ""
```

Comment:
Instead of introducing new variables, we can probably do this:
```suggestion
        if is_rel_path:
        complete_pattern = os.path.join(cephfs.getcwd(), complete_pattern)
    return [path for path in paths if fnmatch.fnmatch(path, complete_pattern)]
```

Reply:
done